command.json

```diff
@@ -428,6 +432,176 @@
               "prompt": false
             }
           ]
+        },
+        {
+          "name": "bootstrap",
+          "description": "Bootstrap GKE cluster",
+          "arguments": [
+            {
+              "name": "environment",
+              "type": "string",
+              "description": "Environment (e.g. staging)"
+            },
+            {
+              "name": "project",
+              "type": "string",
+              "description": "GCloud project id (e.g. bedrock-foundation)",
+              "required": false,
+              "prompt": false
+            }
+          ]
+        },
+        {
+          "name": "provision",
+          "description": "Provision cluster on glcoud with Terraform (subcommands: <plan>, <apply>, <init> and <destroy>)",
+          "commands": [
+            {
+              "name": "plan",
+              "functionName": "terraformPlan",
+              "description": "Terraform plan provisioning of cluster",
+              "arguments": [
+                {
+                  "name": "environment",
+                  "type": "string",
+                  "description": "Environment (e.g. staging)",
+                  "required": false,
```

Comment:
FYI you don't need to list params when they're false but whichever way is nicer to you

Reply:
Yes, I figured that one out later. Was on my list to remove.